compiler/rustc_const_eval/src/check_consts/mod.rs

```diff
@@ -81,7 +82,9 @@ pub fn rustc_allow_const_fn_unstable(
     feature_gate: Symbol,
 ) -> bool {
     let attrs = tcx.hir().attrs(tcx.local_def_id_to_hir_id(def_id));
-    attr::rustc_allow_const_fn_unstable(tcx.sess, attrs).any(|name| name == feature_gate)
+
+    find_attr!(attrs, AttributeKind::AllowConstFnUnstable(syms) => syms.contains(&feature_gate))
+        .unwrap_or(false)
```

Comment:
```rust
find_attr!(attrs, AttributeKind::AllowConstFnUnstable(syms) if syms.contains(&feature_gate))
```

Should work, too, right? without the `unwrap_or(false)`

Reply:
indeed